_book/02-first-steps/05-npm-usage.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Usage with npm"
```

Comment:
How would one decide whether they want to use the node, esm, or cjs methods? There should probably be an introduction to what this is, and what people can expect to find on this page. Right now it's quite detailed in specifics, but with no introduction it makes it hard to understand whether I (as a random reader) need to pay attention to this page.

Reply:
Also, there's no info on this page on how you might include it in a bundler, like Webpack. How would you install verovio through NPM and then configure it to build a bundle, and how would you include that bundle in an HTML page?

Reply:
My repository with the examples is not migrated yet for the final implementation of my pull request https://github.com/rism-digital/verovio/pull/2946, but it's really similar.

Reply:
> I could add them to the verovio repository in ~/emscripten/npm/examples and link to the code examples in this tutorial, what do you think? @lpugin Is there a better place for the examples?

Yes, I think this would be a reasonable place for having them.